toolkits/global/packages/global-card/scss/10-settings/default.scss

```diff
@@ -1,19 +1,16 @@
-@use 'sass:math';
 /**
  * Card
  * Default branding for component
  */
 
-$card--gutter: 16px;
 $card--border: none;
 $card--box-shadow: none;
 $card--font-family: inherit;
 $card--font-size: 14px;
-$card--spacing: math.div($card--gutter, 2);
-$card--padding: $card--gutter;
+$card--spacing: 8px;
```

Comment:
as discussed via the google meets - this should be eventually coming predetermined and defined from tokens - so the developer won't need to type it out by hand.